php-nginx/etc/confd/templates/nginx/site.conf.tmpl

```diff
@@ -21,6 +21,9 @@ server {
     set $access_log_enabled 1;
 
     include /etc/nginx/sites-available/default-*.conf;
+    {{ if ne "" (getenv "WEB_ADDITIONAL_INCLUDES_NAME") }}
+        include /etc/nginx/sites-available/{{ getenv "WEB_ADDITIONAL_INCLUDES_NAME" }}-*.conf;
+    {{ end }}
```

Comment:
I suppose the question is then what in the default-* isn't applicable to the other vhosts (given things like even ssl configuration is in there. Perhaps some thought as to which files from the base image should be always used (or overridable per server)